geoportailv3/templates/index.html

```diff
@@ -196,6 +197,7 @@ <h2 translate>infos</h2>
          appModule.constant('printServiceUrl', "${request.route_url('printproxy')}");
          appModule.constant('shorturlServiceUrl', "${request.route_url('shortener_create')}");
          appModule.constant('defaultExtent', [425152.9429259216, 6324465.99999133, 914349.9239510496, 6507914.867875754]);
+         appModule.constant('maxExtent', [2.6, 47.7, 8.6, 51]);
```

Comment:
for this to work, we would need to set this constant in (appmodule.js)[https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportailv3/static/js/appmodule.js]  ? 

I wanted to keep all constants in index, would make it easier to find stuff later ?